From 3d425c0d50e3dc139f6eed7ed30437aa4139b0c7 Mon Sep 17 00:00:00 2001
From: ryncsn <ryncsn@gmail.com>
Date: Sun, 25 Dec 2016 22:17:51 +0800
Subject: [PATCH 3/5] Unset HAVE_GETPWUID_R for Android

---
 src/util/virutil.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/util/virutil.c b/src/util/virutil.c
index 0291378..1c2091e 100644
--- a/src/util/virutil.c
+++ b/src/util/virutil.c
@@ -52,6 +52,11 @@
 # include <libdevmapper.h>
 #endif
 
+#ifdef __ANDROID__
+/* Android HAVE_GETPWUID_R only works for clang compiler */
+#undef HAVE_GETPWUID_R
+#endif
+
 #include <netdb.h>
 #ifdef HAVE_GETPWUID_R
 # include <pwd.h>
-- 
2.7.3

