From c488136151e6af488a5f7916fa99b1448712d5af Mon Sep 17 00:00:00 2001
From: ryncsn <ryncsn@gmail.com>
Date: Sun, 11 Dec 2016 21:18:03 +0800
Subject: [PATCH] Fix building with android ndk.

---
 byteswap.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/byteswap.h b/byteswap.h
index c7d0cf8..22f3eeb 100644
--- a/byteswap.h
+++ b/byteswap.h
@@ -109,7 +109,7 @@ __bswap_32 (unsigned int __bsx)
 #undef	htonl
 #undef	ntohl
 
-static inline uint32_t
+inline uint32_t
 htonl (x)
      uint32_t x;
 {
@@ -153,7 +153,7 @@ htonl (x)
 #undef	htons
 #undef	ntohs
 
-static inline uint16_t
+inline uint16_t
 htons (x)
      uint16_t x;
 {
-- 
2.7.3

